### PR TITLE
feat: side drawer

### PR DESCRIPTION
**Pull request by susandev6**
1. Use react-navigation-drawer as a solution for the top right side menu button. Removed old solution with SideMenu and SideMenuButton. Added side drawer to each screen (screenshot).
Screen Shot 2020-02-04 at 2 14 00 PM

2. Fixed CustomersResultList and EquipmentResultsList FlatList key issue.

### CODE REVIEW

File: src/components/HeaderButton.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { HeaderButton } from 'react-navigation-header-buttons';
+import Icon from 'react-native-vector-icons/FontAwesome';
```
Comment (svikasg) [inline src/components/HeaderButton.js]: This use of Icon comes from an old iteration - you can instead revert to use `react-native-elements`

Comment (susandev6) [reply src/components/HeaderButton.js]: I am using the Icon for HeaderButton IconComponent. I tried react-native-elements Icon, but got error. Any idea?

Comment (forrestblade) [reply src/components/HeaderButton.js]:
```
<Icon
  color={'grey'}
  name={'mic'}
  type={'feather'}
 />
```
This is how it's being implemented elsewhere without errors?, how ere you using it?

Comment (susandev6) [reply src/components/HeaderButton.js]: I am using it with react-navigation-header-buttons, I need to pass it to the IconComponent.
`<HeaderButton
      {...props}
      IconComponent={Icon}
      iconSize={20}
      color={'#fff'}`

Comment (susandev6) [reply src/components/HeaderButton.js]: I tried to pass in the react-native-elements Icon and it gives me error. :<

Comment (svikasg) [reply src/components/HeaderButton.js]: Can you screenshot the error ?

Comment (susandev6) [reply src/components/HeaderButton.js]: I tried again. It actually complains cannot find the icon with name "th-large" and "times". It seems these two are not supported in react-native-elements?

Comment (susandev6) [reply src/components/HeaderButton.js]: It has been resolved in the updated version

**Pull request merged**